lib/nextflow.config

```diff
@@ -1,2 +1,3 @@
 process.executor = 'sge'
+process.executor.penv = 'smp'
```

Comment:
My test (on DSL2 though) just says "No such variable: penv" unfortunately.

Reply:
Same error on start for DSL1 with a simple test workflow,

Reply:
ah. it seems to be at the same level 
maybe process.pend = 'smp'

Reply:
I'll try that this morning and see.

Reply:
Seems to work!